tchannel/tornado/tchannel.py

```diff
@@ -212,6 +213,12 @@ def request(self,
 
             Defaults to 'c'.
         """
+        # TODO: do not warn if being called from tchannel.TChannel
```

Comment:
The initial planned implementation of `call` will be calling this method. You don't want to tag it as deprecated until the flow is reversed -- the old implementation is calling the new implementation.